applications/MultilevelMonteCarloApplication/python_scripts/adaptive_refinement_utilities.py

```diff
@@ -1,31 +1,33 @@
-from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+# Import Python libraries
+import numpy as np
 
-# Importing the Kratos Library
+# Import Kratos
 import KratosMultiphysics
-
-# Import applications
-import KratosMultiphysics.MeshingApplication as KratosMeshing
-import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
 from KratosMultiphysics.MultilevelMonteCarloApplication.tools import ParametersWrapper
-
-# Import packages
-import numpy as np
-
-"""
-References:
-F. Alauzet, Metric-based anisotropic mesh adaptation, CEA-EDF-INRIA schools: Numerical Analysis Summer School. CEA, Cadarache, France
-Kratos wiki: https://github.com/KratosMultiphysics/Kratos/wiki/MMG-Process
-"""
-
+from KratosMultiphysics.kratos_utilities import CheckIfApplicationsAvailable, IsMPIAvailable
+if KratosMultiphysics.IsDistributedRun():
+    if not IsMPIAvailable():
```

Comment:
Check the mail init, if Kratos is running distributed it is enforced that the mpicore is available, hence this check is not necessary

Reply:
I am not sure I understand your comment. You mean to remove `if not IsMpiAvailable()`?

Reply:
Exactly
See here:
https://github.com/KratosMultiphysics/Kratos/blob/1b59498d2b6e14f3c652d0e5e18123f2721e5c0a/kratos/python_interface/__init__.py#L39-L42

It will crash way earlier hence you dont need this check (I think I saw you using it also in some other places)

Reply:
Done in 84294b7. Thanks.